src/re/ExportImage.re

```diff
@@ -0,0 +1,56 @@
+open Webapi.Dom;
+
+[@bs.module "html2canvas"]
+external html2canvas: (Dom.element) => Js.Promise.t(Dom.element) = "default";
+
+[@bs.scope "window"] [@bs.val]
+external scroll : (int, int) => unit = "scroll";
+
+[@bs.send]
+external toDataURL':
+  (Dom.element, ~type_: string=?, ~encoderOptions: float=?, unit) => string =
+  "toDataURL";
+
+let map = (f) =>
+  fun
+  | Some(v) => Some(f(v))
+  | None => None;
+
+let andThen = (f: 'a => option('b)) =>
+  fun
+  | Some(v) => f(v)
+  | None => None;
+
+let unsafelyUnwrapOption =
+  fun | Some(v) => v
+      | None => raise(Invalid_argument("Passed `None` to unsafelyUnwrapOption"));
```

Comment:
Quick note: 
Take a look here -- https://bucklescript.github.io/bucklescript/api/Belt.Option.html

I'd prefer we use std lib helpers as much as possible!

Reply:
I was able to remove `unsafelyUnwrapOption` but I couldn't remove `andThen` and `map` functions. I think I need a little bit of help here 😅